### PR TITLE
we definetly needs this

### PR DESCRIPTION
**Pull request by mart368b**
So this is why we need to add this to our repository:
- it has 100% test coverage
- it makes it clear what is happening
- it follows state of the art coding techniques and frameworks

### CODE REVIEW

**Comment by emildichmann272** [comment]
I love it, closes all our current issues

**Comment by mart368b** [comment]
But i just made one......

**Comment by emildichmann272** [comment]
Well you are your own demise

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.doesnotwork
```
Comment (Zentarg) [inline .gitignore:1]: I love how you managed to fix all our bugs, as well as the features that don't work in one simple line.

Genius.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+.doessomething
```
Comment (Zentarg) [inline .gitignore:2]: This will soon be an industry standard, I can feel it in my bones. Marvellous.

**Comment by mart368b** [comment]
welp ill just for @Zentarg  help?

**Review by Zentarg — APPROVED** [review]
Overall, I love what you have done to the project, and would greatly appreciate if you continue contributing to this open source project.

**Pull request merged**